### ISSUE TITLE
你好，请问你是如何驱动模型的

### ISSUE THREAD

**Issue by CoderWQYao** [BIGBOSS-dedsec/OpenCV-Unity-To-Build-3DPerson#2]
我尝试使用IK来驱动，但是mediapipe有个问题，他是大腿位是用臀部为原点的，并且原生并没有提供腰部的点位，导致模型腰部和腿部不贴合。
看了你的代码，貌似只有火柴人，希望能分享如何在Unity驱动模型的相关技术。

**Comment by nkligang** [comment]
我也没有找到相关代码，我估计作者不愿意把这部分代码贡献出来

**Comment by yosun** [comment]
yes looking for an efficient way to map ai mocap stuff to different rigs in unity
